python/keras_models.py

```diff
@@ -165,8 +186,17 @@
                                   name="Average_Embeddings")(input_layer)
 
     # Multiply and average
-    mult = keras.layers.Multiply(name="Embeddings_by_Average")([emb1, emb2])
-    avg = keras.backend.mean(mult, axis=2)
+    if RAGGED:
+        # NOTE: I think these are the equivalent ragged-aware ops
+        # but that could be incorrect
+        mult = keras.layers.Lambda(lambda x: tf.math.multiply(x[0], x[1]),
```

Comment:
@scotthlee These are all wrong and should probably be just tf.math.multiply and tf.math.reduce_mean outside of a lambda layer, but I couldn't seem to find the right combination that successfully compiled. Put this on the backburner while we figure out the main model stuff, but we might revisit.

This article seemed to track pretty closely: https://medium.com/tensorflow/introducing-ragged-tensors-ac301c31fd38